src/routes/NewReservationPage.js

```diff
@@ -0,0 +1,119 @@
+import {
+  React, useEffect, useRef, useState,
+} from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { getWorkspaces } from '../redux/workspaces/workspacesSlice';
+import { postReservation } from '../redux/reservations/reservationsSlice';
+
+const NewReservationPage = () => {
```

Comment:
- [ ] I was able to set the end date days before the reservation date (backwards):

![image](https://github.com/Zilola-Nazarova/workspace-reservation-front-end/assets/57421823/bd7dc3aa-d409-4ba3-957e-77022f688842)

Kindly, add some validation so your users can only set the end date after the reservation date. 👍